Source/FieldSolver/FiniteDifferenceSolver/Make.package

```diff
@@ -8,6 +8,7 @@ CEXE_sources += MacroscopicEvolveE.cpp
 #ifdef WARPX_MAG_LLG
 CEXE_sources += MacroscopicEvolveHM.cpp
 CEXE_sources += MacroscopicEvolveHM_2nd.cpp
+CEXE_sources += EvolveHPML.cpp
```

Comment:
We may also need to add this file to the CMakelists file

Reply:
Done.